plugins/rest-plugin/src/main/java/ru/sbtqa/tag/api/manager/EndpointManager.java

```diff
@@ -0,0 +1,67 @@
+package ru.sbtqa.tag.api.manager;
+
+import com.google.common.reflect.ClassPath;
+import java.io.IOException;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.util.HashSet;
+import java.util.Set;
+import org.aeonbits.owner.ConfigFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.sbtqa.tag.api.EndpointEntry;
+import ru.sbtqa.tag.api.annotation.Endpoint;
+import ru.sbtqa.tag.api.context.EndpointContext;
+import ru.sbtqa.tag.api.exception.RestPluginException;
+import ru.sbtqa.tag.api.properties.ApiConfiguration;
+import ru.sbtqa.tag.qautils.errors.AutotestError;
+
+public class EndpointManager {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=81&issues=AWU2KVKaW2AOP0CkD5Fy&open=AWU2KVKaW2AOP0CkD5Fy)

[ISSUE_KEY]: AWU2KVKaW2AOP0CkD5Fy